Formula/osquery.rb

```diff
@@ -122,9 +215,18 @@ def install
     ENV["BOOST_ROOT"] = Formula["osquery"].libexec/"boost/include"
 
     args = std_cmake_args + %W[
-      -Dboost_filesystem_library:FILEPATH=#{libexec}/boost/lib/libboost_filesystem-mt.a
-      -Dboost_regex_library:FILEPATH=#{libexec}/boost/lib/libboost_regex-mt.a
-      -Dboost_system_library:FILEPATH=#{libexec}/boost/lib/libboost_system-mt.a
+      -Daws-cpp-sdk-core_library:FILEPATH=#{libexec}/aws-sdk-cpp/lib/libaws-cpp-sdk-core.a
+      -Daws-cpp-sdk-firehose_library:FILEPATH=#{libexec}/aws-sdk-cpp/lib/libaws-cpp-sdk-firehose.a
+      -Daws-cpp-sdk-kinesis_library:FILEPATH=#{libexec}/aws-sdk-cpp/lib/libaws-cpp-sdk-kinesis.a
+      -Daws-cpp-sdk-sts_library:FILEPATH=#{libexec}/aws-sdk-cpp/lib/libaws-cpp-sdk-sts.a
+      -Dboost_filesystem-mt_library:FILEPATH=#{libexec}/boost/lib/libboost_filesystem-mt.a
+      -Dboost_regex-mt_library:FILEPATH=#{libexec}/boost/lib/libboost_regex-mt.a
+      -Dboost_system-mt_library:FILEPATH=#{libexec}/boost/lib/libboost_system-mt.a
+      -Dcppnetlib-client-connections_library:FILEPATH=#{libexec}/cpp-netlib/lib/libcppnetlib-client-connections.a
+      -Dcppnetlib-uri_library:FILEPATH=#{libexec}/cpp-netlib/lib/libcppnetlib-uri.a
+      -Dlinenoise_library:FILEPATH=#{libexec}/linenoise/lib/liblinenoise.a
+      -Dthrift_library:FILEPATH=#{libexec}/thrift/lib/libthrift.a
```

Comment:
Cool. Might be worth considering a test that runs the application rather than building the library if possible.